yuca/zoo/neuroreflex_glider000.csv

```diff
@@ -0,0 +1,3 @@
+ca_config,neurolife_a.npy
+commit_hash,not_specifiednotes, None
```

Comment:
There could be an error (missing `\n`) with how the notes field is populated....